packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -183,7 +184,7 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, ISuggest
           />) : (null)
         }
         {
-          !moreSuggestionsAvailable && !isMostRecentlyUsedVisible && !isSearching ?
+          isResultsFooterVisible !== false && !moreSuggestionsAvailable && !isMostRecentlyUsedVisible && !isSearching ?
             (<div className={ css('ms-Suggestions-title', styles.suggestionsTitle) }>
               { footerTitle && footerTitle(this.props) }
```

Comment:
It looks like this is already tied to the footer generation on line 128. I'd add isResultsFooterVisible to there and add footerTitle to the check down here.

Reply:
@joschect makes sense. Test needs to be updated since if nothing is specified, the footer div will not be rendered. And it shouldn't be anyways.